Web.md

```diff
@@ -31,25 +31,23 @@ Non-web embeddings are not required to support these additional methods.
 
 :cyclone: Added for milestone 2, developers must feature detect.
 
-In Web embeddings, the following overloads are added (in addition to the core
-JS API method of the same name).
+In Web embeddings, the following methods are added.
 
 ```
-Promise<WebAssembly.Module> compile(Response source)
-
-Promise<WebAssembly.Module> compile(Promise<Response> source)
+Promise<WebAssembly.Module> compileStreaming(source)
 ```
 
-Developers can set the argument `source` with either a promise that resolves
-with a
+`source` is unconditionally passed through `Promise.resolve(source)`.
```

Comment:
You'll want to add "using the built-in value of Promise.resolve" just so that nobody thinks that monkeypatching `Promise.resolve = function () { }` should affect this implementation.